apps/website-25/src/components/Nav.tsx

```diff
@@ -146,9 +171,22 @@ export const isCurrentPath = (url: string): boolean => {
 export const Nav: React.FC<NavProps> = ({
   className, logo, courses, primaryCtaText, primaryCtaUrl,
 }) => {
-  const [expandedSections, setExpandedSections] = useState<'none' | 'nav' | 'nav-and-explore'>('none');
-  const navExpanded = expandedSections === 'nav' || expandedSections === 'nav-and-explore';
+  const [expandedSections, setExpandedSections] = useState<'none' | 'nav' | 'nav-and-explore' | 'profile'>('none');
```

Comment:
do you think this renaming would help with clarity?
```suggestion
  const [expandedSections, setExpandedSections] = useState<'none' | 'mobile-nav' | 'explore' | 'profile'>('none');
```

Reply:
Yep I agree that's better, changed now